### ISSUE TITLE
[Synthèse] - Afficher le ou les zonages lors des requêtes par entité géographique

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/GeoNature#3176]
Dans la synthèse, on peut requêter par entité géographique. 

Lorsqu'on importe un zonage geojson, l'objet s'affiche sur la carte. 

Il serait intéressant lorsqu'on recherche à partir du ref_geo d'afficher sur la carte le ou les items sur lesquels se fait la requête (notamment pour savoir si une absence de point correspond à une zone blanche de notre site ou simplement parce qu'on est en dehors de notre site).

**Comment by camillemonchicourt** [comment]
Sinon tu peux ajouter des couches du ref_geo affichables sur la carte depuis le gestionnaire de couches.

**Comment by DonovanMaillard** [comment]
oui, mais par exemple afficher la couche commune est lourd sur la métro, afficher la ou les 2-3 communes sur lesquelles on fait une requête ne présente pas ce souci de lourdeur, et me semblerait cohérent

**Comment by camillemonchicourt** [comment]
Ouais ouais mais ça fait un dev et un comportement assez spécifique uniquement pour un des critères de recherche, donc pas hyper simple ni générique.
Mais certainement faisable oui.
